userprofile/models.py

```diff
@@ -203,6 +222,19 @@ def update_from_healthcheck(self, healthcheck: Covid19Triage) -> None:
             if has_value(v):
                 self.data[k] = v
 
+    def update_post_screening_study_arms(self):
+        if not self.hcs_study_a_arm and settings.HCS_STUDY_A_ACTIVE:
+            self.hcs_study_a_arm = self.get_random_study_arm()
+            update_turn_contact.delay(
+                self.msisdn, "hcs_study_a_arm", self.hcs_study_a_arm
```

Comment:
Just want to check, will this be a string value, or does django return some other type for `TextChoices`? ie. do you need to do a `self.hcs_study_a_arm.value` or something.